app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  post.update(likes_counter: post.likes_counter + 1)
```

Comment:
- [ ] The like update counter should be defined in a private method for the like model , kindly fix it 🙏 
example : 

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```